src/main/java/com/pscheduler/server/model/GPACourse.java

```diff
@@ -0,0 +1,40 @@
+package com.pscheduler.server.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Version;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+
+@Entity
+public class GPACourse {
+
```

Comment:
Don't forget the private variable changes

Reply:
`crn`, `courseNumber`, `name`, `instructor`, `credits`, `gpa`, `students`, `A`, `B`, `C`, `D`, `F`, `withdraws`

Reply:
Done